### PR TITLE
updated readme

### PR DESCRIPTION
**Pull request by hamzeh-p**
I thought it would be nicer to bold the **get_post()** method in readme file.
I also changed little snippet to snippet

### CODE REVIEW

**Comment by hamzeh-pourshabanan** [comment]
Thank you for your contribution

**Comment by hamzeh-pourshabanan** [comment]
> Thank you for your contribution

@ryne-zialu your changes will be merged soon

**Comment by hamzeh-p** [comment]
> > Thank you for your contribution
> 
> @ryne-zialu your changes will be merged soon

that's very nice of you

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # wordpress-transient
 
-In this little snippet of code we can see how to cache the get_posts() in wordpress,
+In this snippet of code we can see how to cache the **get_posts()** in wordpress,
```
Comment (hamzeh-pourshabanan) [inline README.md]: wouldn't it be nicer to replace little with tiny?

Comment (hamzeh-p) [reply README.md]: you're right, thanks for feed back, I'll commit it right now

**Review by hamzeh-pourshabanan — APPROVED** [review]

**Comment by ryne-zialu** [comment]
[google](google.com)

**Pull request merged**